ansible/validate.yml

```diff
@@ -2,6 +2,23 @@
 
 # Fail early if configuration is invalid
 
+- name: Validate podman configuration
```

Comment:
Put in podman role?

Reply:
See comment above as to why all of this isn't in the role.